### DISCUSSION TITLE
v3.36 release?

### DISCUSSION THREAD

**Discussion by Xetera** [qdm12/gluetun#1876] [Q&A]
I'm currently using gluetun as an addon in truecharts which requires a release and won't allow you to specify a specific container name (to the best of my knowledge).

v3.36 allows port forwarding on protonvpn which is quite useful for me so I'd love to see this in a new release I can pull

**Comment by broken4214** [comment]
Seconded, patiently waiting for 3.36 to release so Truecharts can pull it upstream. Slightly annoyed that protonvpn port forwarding was added to the wiki with no where mentioning that it wasn't released in a mainline release yet.
